diff --git a/tokio/tests/task_local_set.rs b/tokio/tests/task_local_set.rs
index a70f49b81a..f8a35d0ede 100644
--- a/tests/task_local_set.rs
+++ b/tests/task_local_set.rs
@@ -16,16 +16,6 @@ use std::sync::atomic::Ordering::{self, SeqCst};
 use std::sync::atomic::{AtomicBool, AtomicUsize};
 use std::time::Duration;
 
-#[tokio::test(flavor = "current_thread")]
-async fn localset_implicit_current_thread() {
-    task::spawn_local(async {}).await.unwrap();
-}
-
-#[tokio::test(flavor = "multi_thread")]
-async fn localset_implicit_multi_thread() {
-    task::spawn_local(async {}).await.unwrap();
-}
-
 #[tokio::test(flavor = "current_thread")]
 async fn local_basic_scheduler() {
     LocalSet::new()